File: Cc.Core/Models/OrderRow.cs
using System;
using System.Collections.Generic;

namespace Cc.Core.Models
{
    public partial class OrderRow
    {
        public long Id { get; set; }
        public long OrderId { get; set; }
        public long ProductId { get; set; }
        public int Quantity { get; set; }
        public long MeasureId { get; set; }
        public long? WarehouseId { get; set; }
        public string? RegistrationNumber { get; set; }
        public string? InsertUser { get; set; }
        public DateTime? InsertDate { get; set; }
        public string? UpdateUser { get; set; }
        public DateTime? UpdateDate { get; set; }

        public virtual Measure Measure { get; set; } = null!;
        public virtual Product Product { get; set; } = null!;
    }
}

File: Cc.Ui/src/models/orderRow.ts
import { Entity, resource, field, required } from '../framework/entity';
import { Pagination, Pager } from '../framework/core/actions';

@resource('orderRow')
export class OrderRow extends Entity {

    @field()
    id: number;
    @field()
    orderId: number;
    @field()
    productId: number;
    @field()
    quantity: number;
    @field()
    measureId: number;
    @field()
    warehouseId: number;
    @field()
    registrationNumber: string;
    @field()
    insertUser: string;
    @field()
    insertDate: string;
    @field()
    updateUser: string;
    @field()
    updateDate: string;

    constructor(data?: any) {
        super()
        this.init(data);      
    }

    static newItem(): OrderRow {

        let item = new OrderRow();
        return item;
    }
}

@resource('orderRow')
export class OrderRowSearchModel extends Entity implements Pagination {
    
    @field()
    orderId?: number;
    @field()
    productId?: number;
    @field()
    quantity?: number;
    @field()
    measureId?: number;
    @field()
    warehouseId?: number;
    @field()
    registrationNumber?: string;

    @field()
    pager: Pager = <Pager> {
        take: 100,
        orderBy: 'id'
    };
}